extension/src/artillery/mod.rs

```diff
@@ -1,18 +1,68 @@
 use std::f64::consts::{FRAC_PI_4, PI};
 
-use arma_rs::Group;
+use arma_rs::{Context, Group};
+use rayon::prelude::*;
 
 use crate::common::{Temperature, GRAVITY};
 
 mod simulate;
 
 pub fn group() -> Group {
-    Group::new()
+    Group::new().command("calculate_table", calculate_table)
 }
 
-fn findMaxAngle(muzzel_veocity: f64, air_friction: f64) -> (f64, f64) {
+fn calculate_table(
+    ctx: Context,
+    muzzle_velocity: f64,
+    air_friction: f64,
+    min_elev: f64,
+    max_elev: f64,
+    high_arc: bool,
+) -> (f64, u32) {
+    let mut min_elev = ((PI / 180.0) * min_elev).max(2.0 * (PI / 180.0)); // cap min to 2 degrees (negative elev might get messy)
+    let mut max_elev = ((PI / 180.0) * max_elev).max(88.0 * (PI / 180.0)); // cap max to 88 degrees (mk6)
```

Comment:
Since `(PI / 180.0)` will be calculated many times, shouldn't shouldn't it be done once ?

Reply:
any (even slightly) smart compiler will do those calculations for you during compile time

Reply:
Ok 👍 That'll be great.